src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleEqualityTests.cs

```diff
@@ -1358,12 +1358,9 @@ static void Main()
                 // (6,30): error CS0034: Operator '==' is ambiguous on operands of type '<null>' and 'default'
                 //         System.Console.Write((null, () => 1) == (default, default));
                 Diagnostic(ErrorCode.ERR_AmbigBinaryOps, "(null, () => 1) == (default, default)").WithArguments("==", "<null>", "default").WithLocation(6, 30),
-                // (6,37): error CS8773: Feature 'inferred delegate type' is not available in C# 9.0. Please use language version 10.0 or greater.
+                // (6,30): error CS0019: Operator '==' cannot be applied to operands of type 'lambda expression' and 'default'
```

Comment:
I wonder if later on (i.e. not in this PR/release) it will be practical to devise some way to still give LangVersion diagnostics. For example, by checking again at the point we give a conversion error if a valid conversion "would have" existed in the later language version.